lab_2/main.py

```diff
@@ -91,6 +91,8 @@ def find_lcs_length(first_sentence_tokens: tuple, second_sentence_tokens: tuple,
         return -1
     if not first_sentence_tokens or not second_sentence_tokens:
         return 0
+    if len(first_sentence_tokens) > len(second_sentence_tokens):
```

Comment:
do you need this check?

Reply:
i think i should have this so the code can handle different sized token inputs

Reply:
Ok